docker/images/runners/n8n-task-runners.json

```diff
@@ -0,0 +1,53 @@
+{
+	"task-runners": [
+		{
+			"runner-type": "javascript",
+			"workdir": "/home/runner",
+			"command": "/usr/bin/node",
+			"args": [
+				"--disallow-code-generation-from-strings",
+				"--disable-proto=delete",
+				"/opt/runners/task-runner-javascript/dist/start.js"
+			],
+			"health-check-server-port": "5681",
+			"allowed-env": [
+				"PATH",
+				"GENERIC_TIMEZONE",
+				"N8N_RUNNERS_GRANT_TOKEN",
+				"N8N_RUNNERS_TASK_BROKER_URI",
+				"N8N_RUNNERS_MAX_PAYLOAD",
+				"N8N_RUNNERS_MAX_CONCURRENCY",
+				"N8N_RUNNERS_TASK_TIMEOUT",
+				"N8N_RUNNERS_HEARTBEAT_INTERVAL",
+				"N8N_RUNNERS_HEALTH_CHECK_SERVER_ENABLED",
+				"N8N_RUNNERS_HEALTH_CHECK_SERVER_HOST",
+				"N8N_RUNNERS_HEALTH_CHECK_SERVER_PORT",
+				"NODE_FUNCTION_ALLOW_BUILTIN",
+				"NODE_FUNCTION_ALLOW_EXTERNAL",
+				"NODE_OPTIONS",
+				"NODE_PATH",
+				"N8N_SENTRY_DSN",
+				"N8N_VERSION",
+				"ENVIRONMENT",
+				"DEPLOYMENT_NAME"
+			]
+		},
+		{
+			"runner-type": "python",
+			"workdir": "/home/runner",
+			"command": "/opt/runners/task-runner-python/.venv/bin/python",
+			"args": ["-m", "src.main"],
+			"health-check-server-port": "5682",
+			"allowed-env": [
+				"PATH",
+				"GENERIC_TIMEZONE",
```

Comment:
The Python runner doesn't have access to the workflow data proxy so I think there's no need for `GENERIC_TIMEZONE`.

Reply:
Removed